src/utils/helpers.ts

```diff
@@ -0,0 +1,86 @@
+import { Person, Title } from '../types';
+import { PersonSex } from '../types/enum';
+
+const getFieldFromTitle = (column:Title, columnCode:string) => {
+  switch (columnCode) {
+    case 'name':
+      return column.name;
+
+    case 'sex':
+      return column.sex;
+
+    case 'born':
+      return column.born;
+
+    case 'died':
+      return column.died;
+
+    default:
+      return null;
+  }
+};
+
+export const getFilteredPeople = (
+  people: Person[],
+  query: string,
+  sortBySex: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+): Person[] => {
```

Comment:
The function looks too complicated, split logic, for example, one function for filter and the second for sorting